db-engine/src/main/java/com/splicemachine/db/iapi/sql/dictionary/DataDictionary.java

```diff
@@ -245,6 +244,9 @@
     int SYSSNAPSHOT_NUM=32;
     int SYSTOKENS_NUM=33;
     int SYSREPLICATION_CATALOG_NUM=34;
+    int SNAPAPPL_CATALOG_NUM=35;
```

Comment:
only system table needs catalog num, not system views, I think we need to give a catalog number for SYSIBMADM.MON_GET_CONNECTION, but not the views inside it.

Reply:
Ah, now I get it. Youssef and I had this exact question about what does the system do when creating non-core system tables, this MON_GET_CONNECTION is concurrently being created by three threads (due to three entries of views I added).